example.config.toml

```diff
@@ -1,6 +1,11 @@
+# All fields are required for now
 [general]
+token_size = 123
 # Telegram ID of the master account
 masterid = 1234578
```

Comment:
Oversight by myself but the default id should be 777000 so the check inside the main function makes sense